src/tools/launcher/applauncherwidget.cpp

```diff
@@ -112,7 +114,7 @@ void AppLauncherWidget::launch(const QModelIndex &index) {
                                tr("Unable to launch in terminal."));
         }
     } else {
-        QProcess::startDetached(command);
+         QProcess::startDetached(app_name,{m_tempFile});
```

Comment:
is this whitespace intentional?

Reply:
No just a typo. When we pick a clang-format profile that will automatically fix stuff like this.